CONTRIBUTING.md

```diff
@@ -389,27 +389,41 @@ $ yarn watch & # runs in the background
 * Make sure to update the PR title/description if things change. The PR title/description are going to be used as the
   commit title/message and will appear in the CHANGELOG, so maintain them all the way throughout the process.
 
+#### Adding dependencies bundled in lambda layers
+
+Sometimes, constructs introduce runtime dependencies bundled in lambda layers for use in custom resources.
+The CDK includes three of these as direct dependencies for legacy purposes: `@aws-cdk/asset-awscli-v1`,
+`@aws-cdk/asset-kubectl-v20`, and `@aws-cdk/asset-node-proxy-agent-v5`. The actual dependencies are pulled out
+to separate repositories (like this [one](https://github.com/cdklabs/awscdk-asset-kubectl) for kubectl)
+and managed there. If you would like to introduce additional runtime dependencies, it likely involves discussing
+with a CDK maintainer and opening a new repository in cdklabs that vends the lambda layer. Generally, each branch
```

Comment:
[nit] Do we have a public list that we could point to?

Reply:
We don't have one, and it would be too annoying to maintain :/ I do agree, it would be nice though.